FOSSGIS/2017/workshop_notes.md

```diff
@@ -0,0 +1,196 @@
+# Preparation step
+
+## Downloading OSM data
+
+
+http://download.geofabrik.de/europe/germany/bayern/niederbayern.html
+
+```sh
+mkdir -p ~/data && cd ~/data
+wget http://download.geofabrik.de/europe/germany/bayern/niederbayern-latest.osm.pbf
+```
+
+*** Setting up the osrm-backend Docker image
+
+https://hub.docker.com/r/osrm/osrm-backend-ci/
+
+```sh
+docker pull osrm/osrm-backend-ci
```

Comment:
I think pull is not required, the first docker run cmd below will fetch the image automatically